src/main/java/com/internet/shop/controllers/LoginController.java

```diff
@@ -31,7 +30,7 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
         String password = req.getParameter("password");
         try {
             User user = authenticationService.login(login, password);
-            session.setAttribute(USER_ID, user.getId());
+            session.setAttribute(AddProductToCartController.USER_ID, user.getId());
```

Comment:
I would move the original constant to LoginController

Reply:
Done